user/caching.md

```diff
@@ -206,10 +209,11 @@ If none of the previous locations contain a valid cache, the build continues wit
 
 After the first pull request build is run, it creates a new pull request cache.
 
-Two important things to note about caching for pull requests:
+Some important things to note about caching for pull requests:
 
 * If a repository has *Build pushes* set to *OFF*, neither the target branch nor the master branch can ever be cached.
-* If the cache found is old, for example in a workflow where most work happens on branches, the less useful the cache will be.
+* If the cache on the master branch is old, for example in a workflow where most work happens on branches, the less useful the cache will be.
+* If a pull request has a cache, to clear that cache you need to clear **both** the pull request cache **and** the cache of the target branch.
```

Comment:
maybe we could say "to not use a cache"? 

to clear the PR cache you clear just the pull request cache, but to not fetch any cache, you'd need to clear both, does that make sense?